### PR TITLE
feat: improvements on WAI and related standards/recommendations

### PR DESCRIPTION
**Pull request by PassionPenguin**
### Description

- **fixed an outdated entry - WCAG**
- improvements on existing WAI, ATAG, WCAG, ARIA glossary
- added UAAG entry, part of the Accessibility Guidelines series purposed by WAI

Relates to #34143

### CODE REVIEW

**Comment by PassionPenguin** [comment]
@hamishwillee apart from simply rewriting things, the PR itself contains commits that **fixes outdated WCAG entry**, **adds UAAG entry**, part of the Accessibility Guidelines series purposed by WAI.

File: files/en-us/glossary/wai/index.md [hunk 1]
```diff
@@ -6,9 +6,18 @@ page-type: glossary-definition
 
 {{GlossarySidebar}}
 
-WAI or Web Accessibility Initiative is an effort by the World Wide Web Consortium (W3C) to improve accessibility for people with various challenges, who may need a nonstandard {{Glossary("browser")}} or devices.
+WAI or Web {{glossary("Accessibility")}} Initiative is an effort by the {{glossary("W3C")}} to develop standards and guidelines to help everyone build a web by improving accessibility for people with various challenges.
```
Comment (estelle) [inline files/en-us/glossary/wai/index.md:9]:
```suggestion
The **Web Accessibility Initiative (WAI)** is an effort by the {{glossary("W3C")}} to develop standards and guidelines to help everyone building the web ensure and improve {{glossary("accessibility")}} for people with various challenges.
```
* Move the "accessiblity" macro link out of the names of organizations, making the next occurrence of the link the 
* The first mention of the org should be the full name, followed by the acronym; then the acronym can be used freely.

File: files/en-us/glossary/wai/index.md [hunk 2]
```diff
@@ -6,9 +6,18 @@ page-type: glossary-definition
...
+
+WAI develops guidelines and other technical reports through the same process as other parts of the W3C. Accessibility guidelines developed include web content ({{glossary("WCAG")}}), authoring tools ({{glossary("ATAG")}}), and user agents ({{glossary("UAAG")}}). WAI also develops the Accessible Rich Internet Applications ({{Glossary("ARIA")}}) suite of technologies to make Web content and Web applications more accessible to people with disabilities. It especially helps with dynamic content and advanced user interface controls developed with HTML, JavaScript, and related technologies.
```
Comment (estelle) [inline files/en-us/glossary/wai/index.md:11]:
```suggestion
WAI develops guidelines and other technical reports through the same process as other parts of the W3C. WAI develops accessibility guidelines for web content ({{glossary("WCAG")}}), authoring tools ({{glossary("ATAG")}}), and user agents ({{glossary("UAAG")}}). WAI also develops the Accessible Rich Internet Applications ({{Glossary("ARIA")}}) suite of technologies to make Web content and Web applications more accessible to people with disabilities. It especially helps with dynamic content and advanced user interface controls developed with HTML, JavaScript, and related technologies.
```

File: files/en-us/glossary/wcag/index.md [hunk 3]
```diff
@@ -6,22 +6,26 @@ page-type: glossary-definition
 
 {{GlossarySidebar}}
 
-_Web Content Accessibility Guidelines_ (**WCAG**) are a recommendation published by the {{Glossary("WAI","Web Accessibility Initiative")}} group at the {{Glossary("W3C")}}. They outline a set of guidelines for making content accessible primarily for people with disabilities but also for limited-resource devices such as mobile phones.
+_Web Content {{glossary("Accessibility")}} Guidelines_ (**WCAG**) is a recommendation published by the {{Glossary("WAI","Web Accessibility Initiative")}} group at the {{Glossary("W3C")}}, outlining a set of guidelines for making content accessible primarily for people with disabilities but also for limited-resource devices such as mobile phones.
```
Comment (estelle) [inline files/en-us/glossary/wcag/index.md:9]:
```suggestion
_Web Content Accessibility Guidelines_ (**WCAG**) is a recommendation published by the {{Glossary("WAI","Web Accessibility Initiative (WAI)")}} group at the {{Glossary("W3C")}}, outlining a set of guidelines for making content {{glossary("accessible")}} primarily for people with disabilities but also for limited-resource devices such as mobile phones.
```

File: files/en-us/glossary/wcag/index.md [hunk 4]
```diff
@@ -6,22 +6,26 @@ page-type: glossary-definition
...
 
-WCAG uses three levels of conformance:
+WCAG 2 uses three levels of conformance:
 
 - Priority 1: Web developers **must** satisfy these requirements, otherwise it will be impossible for one or more groups to access the Web content. Conformance to this level is described as A.
 - Priority 2: Web developers **should** satisfy these requirements, otherwise some groups will find it difficult to access the Web content. Conformance to this level is described as AA or Double-A.
 - Priority 3: Web developers **may** satisfy these requirements, in order to make it easier for some groups to access the Web content. Conformance to this level is described as AAA or Triple-A.
 
-Work is proceeding on both [WCAG 2.2](https://www.w3.org/TR/WCAG22/) and [WCAG 3.0](https://www.w3.org/TR/wcag-3.0/).
+[WCAG 2.2](https://www.w3.org/TR/WCAG22/) was published on Oct, 2023 and [WCAG 3.0](https://www.w3.org/TR/wcag-3.0/) is now an early draft.
```
Comment (estelle) [inline files/en-us/glossary/wcag/index.md:19]:
```suggestion
[WCAG 2.2](https://www.w3.org/TR/WCAG22/) was published on Oct, 2023 and [WCAG 3.0](https://www.w3.org/TR/wcag-3.0/) is in development.
```

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Pull request merged**